test/scripts/docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: '3'
 services:
   localstack:
     container_name: local_sqs_aws
-    image: localstack/localstack:1.2.0@sha256:41216e34e25546654c58b8075b133027eab3d958e605ef2e7af168d1f05a9364
-    command: sh -c "/usr/local/bin/init.sh && exec myexecutable"
+    image: localstack/localstack:latest@sha256:da35c19b83b646b2f4511b62f39689bc73cf0fe53ac4159dfd513e9774356324
```

Comment:
I believe localstack will soon release a fixed version of their Docker image, so we may be better off waiting for that, I'd prefer not to use latest.

Reply:
As we were discussing this, they just released v3 of localstack alongside a 'stable' and '3' tag.
I can update the MR to use localstack/localstack:stable or localstack/localstack:3 instead.
Let me know if you want me to do so.

Reply:
I've just checked with both 'stable' and '3' tagged releases of localstack, integration tests pass; I await your decision for this.

Reply:
Committed a change to fix the version just now.